### PR TITLE
add DIFT

### PR DESCRIPTION
**Pull request by souravjain540**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ It has instructions on how to get started with the process, which makes it reall
 
 https://github.com/firstcontributions/first-contributions
 
+------
+
+## Dozen Issues for First Timers
+
+> To help you get started Amplication have found a Dozen issues for first-time contributors, that can be a great way to fire-start your journey of the open-source contribution.
```
Comment (ykdojo) [inline README.md:23]: Dozen -> dozen

Amplication: explain what it is?

File: README.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ It has instructions on how to get started with the process, which makes it reall
...
+
+By giving back to the community, you’re also getting a chance to win cool limited-edition swag.
```
Comment (ykdojo) [inline README.md:25]: Let's make it less sales-y here.

I'd just say, "win some swag"

File: README.md (hunk 1 above)
Comment (ykdojo) [inline README.md:23]: Can you also specify the timeline for this?

**Review by ykdojo — CHANGES_REQUESTED** [review]
First round! Thank you

**Comment by souravjain540** [comment]
done @ykdojo

**Pull request closed without merging**